src/heroes/doomfist/slam.opy

```diff
@@ -110,18 +110,20 @@ rule "[doomfist/slam.opy]: Execute slam":
             waitUntil(eventPlayer.isOnGround() or not eventPlayer.isUsingAbility1(), Math.INFINITY)
             cancelMomentum(eventPlayer)
             eventPlayer.stopForcingThrottle()
+            wait(0.25)
```

Comment:
```suggestion
```
since the 0.25 wait is in both if and else if, pull it outside the conditional at the end